src/middlewares/auth.js

```diff
@@ -67,7 +67,10 @@ class Authentication {
    */
   static decodeJwt(token) {
     let payload = null;
-    payload = jwt.decode(token, index.secret);
+    payload = jwt.verify(token, index.secret, (err, decoded) => {
+      if (err) return false;
+      return decoded;
+    });
```

Comment:
You also changed this snippet. Is there a reason why?

Reply:
The jwt method ```decode``` was not performing any checks for secrets and as such was allowing for manipulations and alteration of the token. Running tests while it was on ```jwt-decode``` allowed for the passage of these altered tokens